profiles/source_profile.go

```diff
@@ -409,13 +410,91 @@ func NewSourceProfileCsv(params map[string]string) SourceProfileCsv {
 	return csvProfile
 }
 
+type SourceProfileStreamingMySQL struct {
```

Comment:
Why do we need a new profile for this, and not just add StreamingConfig to the existing profile ?

Reply:
I kept it different because of separate modes. MOdified to just add the cfg to the older connection profiles.